README.md

```diff
@@ -20,6 +20,9 @@ As of writing this on 2024-05-04, nixos-unstable seems to come with nix 2.18.2
 nix build github:BatteredBunny/brew-nix#blender
 ./result/Applications/Blender.app/Contents/MacOS/Blender
 ```
+## Using with Flakes stand-alone
```

Comment:
```suggestion
## Using with nix-darwin
```
Should mention that its for nix-darwin imo

Reply:
That makes sense. Integrated your request by rebasing and force pushing, so the number of commits stays the same.